localization/english/dvg_flavor_lotharingia_l_english.yml

```diff
@@ -0,0 +1,100 @@
+﻿l_english:
+ #Journal Entries
+ je_lotharian_house_of_cards: "Lotharingia's House of Cards"
+ je_lotharian_house_of_cards_reason: "Under the regime established by De Aigle, Lotharingia sits atop a 'House of Cards' as some might call it, for how fragile it is. If reforms are not made, it is possible for a civil war to be inevitable."
+ je_reconquering_atlantia: "Reclaiming Cote du Cafe"
+ je_reconquering_atlantia_reason: "Cote du Cafe was once our province, but taken away from us. Lotharingia is not complete without it."
+ je_lotharian_house_of_cards_goal: "If Lotharingia stays a dictatorship under these conditions long enough, it is doomed to face another civil war."
+ 
+je_reuniting_lotharingia: "Reuniting Lotharingia"
+je_reuniting_lotharingia_reason: "After having been backstabbed by people who once called themselves Lotharians, we must reassemble the house of cards. As a mythical person once said, 'A house divided against itself cannot stand.'"
+
+je_odonia_unite: "Liberating Lotharingia"
+je_odonia_unite_reason: "With many turning against the old regime, it is up to us to bring order to what is left of the country."
+
+je_guarani_unite: "Retaking Guarani Lands"
+je_guarani_unite_reason: "Now that we can act as an independent nation, we should refocus on being able to regain our homelands."
+
+je_lot_sol: "Raising Lotharingia's Standard of Living"
+je_lot_sol_reason: "Although the main point of contention is De Aigle's cronyism, civil war progress can be delayed if the standard of living can be raised."
+ #Events
+ 
+reconquering_atlantia.1.t: "The Return of the Golden Province"
+reconquering_atlantia.1.d: "Thanks to our might, we have managed to retake Cote du Cafe."
+reconquering_atlantia.1.f: "A land rich in gold mines and coffee plantations, imagine all the wealth it can bring us! It seems like nothing beats the power of threats and violence if you don't get your way, we would know very well of course."
+reconquering_atlantia.1.a: "Another card for our house!"
+
+dvg_lotharingia.1.t: "The Republic of Lotharingia"
+dvg_lotharingia.1.d: "After fighting not only for its independence, but for the civil war that ensued right after it, Lotharingia now finds itself under the grip of war hero and despot Denis de Aigle, along with his cronies."

```

Comment:
Instead of hardcode leader 'Denis de Aigle' use [ROOT.GetCountry.GetRuler.GetFullName]
This will also give context.
Sam for Lotharingia , use [ROOT.GetCountry.GetName]